src/huntsman/pocs/observatory.py

```diff
@@ -207,7 +205,7 @@ def take_flat_fields(self, camera_names=None, alt=None, az=None,
             safety_func = partial(self.is_dark, horizon='flat')
 
         # Load the flat fielding config
-        flat_field_config = self.get_config('flat_fields')
+        flat_field_config = self.get_config('flat_fields', default=dict())
```

Comment:
I think we have some inconsistency between `flat_fields` and `flat_field`. They are changed here to the `flat_fields` but the config still has `flat_field`.

Reply:
Yep. I've changed it for the testing config but still need to update the huntsman config.